extensions/arc/deployment/src/main/java/io/quarkus/arc/deployment/ArcConfig.java

```diff
@@ -59,6 +59,9 @@
      * If set to true, the bytecode of unproxyable beans will be transformed. This ensures that a proxy/subclass
      * can be created properly. If the value is set to false, then an exception is thrown at build time indicating that a
      * subclass/proxy could not be created.
+     *
+     * Furthermore when enabled, this setting allows Quarkus to properly create regular scoped class beans whose methods
```

Comment:
I was more thinking about a list of all transformations we do... final class, no-args constructors, final methods of bean classes (for proxies/subclasses). This sentence is more or less redundant.

Reply:
OK, fixed